src/commcare_cloud/ansible/roles/deploy_hq/tasks/staticfiles_compress.yml

```diff
@@ -6,7 +6,6 @@
   when: not stop_loop_hack.failed | default(False)
   loop:
```

Comment:
I wonder if this loop could be removed, but I didn't investigate and decided to just err on the side of making a smaller change.